CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/parameters/types/StringParameterType.java

```diff
@@ -139,10 +139,16 @@ public static boolean isLabel(final PluginParameter<?> parameter) {
     public static void setIsLabel(final PluginParameter<?> parameter, final boolean isLabel) {
         parameter.setProperty(IS_LABEL, isLabel);
     }
-
+    /**
+     *
+     * @param param
+     * @param stringValue
+     * @return null when the string is valid, will return "parameter is Empty!"
+     * when invalid
+     */
     @Override
     public String validateString(PluginParameter<StringParameterValue> param, String stringValue) {
-        if (StringUtils.isNotBlank(stringValue) && stringValue.trim().isEmpty()) {
+        if (StringUtils.isBlank(stringValue)) {
```

Comment:
Similar to the `PasswordParameterType` can you please update `CHANGELOG.md` with the change in behaviour please @Atlas139mkm ?

Reply:
same as my comment above

Reply:
also see my above comment